apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -1,33 +1,129 @@
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import color from "@cdo/apps/util/color";
-import SectionRow from './SectionRow';
+import {Table, sort} from 'reactabular';
 import i18n from '@cdo/locale';
 import { styles as tableStyles } from '@cdo/apps/templates/studioHomepages/SectionsTable';
```

Comment:
is this shared styling still applicable? If not, we can nix the import and I think change the export and comment referencing it in `SectionsTable`.